pkg/healthcheck/healthcheck_multicluster.go

```diff
@@ -143,6 +144,14 @@ func (hc *HealthChecker) multiClusterCategory() category {
 					return hc.checkRemoteClusterAnchors()
 				},
 			},
+			{
+				description: "multicluster daisy chaining is avoided",
```

Comment:
TIOLI: I think we can rename that to something that does not imply that this is an active step that avoids daisy chaining such as: "no multicluster daisy chaining"

Reply:
For clarity and consistency, checks should always be stated in the affirmative.  i.e. they should never start with "no".

Reply:
Makes sense